File: 2580-CircularSentence/2580-CircularSentence.js
// Last updated: 6/22/2026, 12:50:56 PM
/**
 * @param {string} sentence
 * @return {boolean}
 */
var isCircularSentence = function (sentence) {
    let sentLen = sentence.length
    for(let i=0; i<sentLen; i++){
        if(sentence[i] != ' ') continue;
        if(sentence[i-1] != sentence[i+1]) return false;
    }
    return (sentence[0] == sentence[sentLen - 1]);
};

File: 2580-CircularSentence/2580-CircularSentence.c
// Last updated: 6/22/2026, 12:51:03 PM
bool isCircularSentence(char* sentence) {
    int i=0;
    for(i; sentence[i] != '\0'; i++){
        if((sentence[i] != ' ')) continue;
        if(sentence[i-1] != sentence[i+1]) return false;
    }
    return (sentence[0] == sentence[i - 1]);
}